apps/prairielearn/src/lib/group-update.ts

```diff
@@ -222,6 +228,7 @@ export async function randomGroups({
               group_name: null,
               uids: users,
               authn_user_id,
+              authzData: dangerousFullAuthzForTesting(),
```

Comment:
Is there a reason that `randomGroups(...)` wasn't updated to take in an `authzData` object like `uploadInstanceGroups(...)` was?

Reply:
That is not true:

https://github.com/PrairieLearn/PrairieLearn/blob/5247c3b0ce8105e4378da2e3921d22cf2ac234ec/apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ts#L105-L114